server/services/profanityCheck.js

```diff
@@ -0,0 +1,6 @@
+const fs = require('fs');
+const wordListPath = require('word-list');
+const validWords = fs.readFileSync(wordListPath, 'utf8').split('\n');
+
+const isValidWord = word => validWords.some(validWord => validWord.toLowerCase() === word.toLowerCase());
```

Comment:
would it be worth making the validWords file all lowercase and saving computation time? For every word that's being checked your going to have to lowercase every word in your dictionary

Reply:
good idea, ill add that!